README.md

```diff
@@ -84,6 +84,10 @@ rows: computed(function() {
 });
 ```
 
+### Using ember-table with Ember <= 3.24
+Use resolutions in your package.json to pin down `ember-classy-page-object` to version `0.7.0`.
+Newer versions are used to support Ember >= 3.28
+
```

Comment:
I'd suggest we move this up higher in the README to be included with the installation instructions.